MaxText/tests/integration_tests/checkpointing_test.py

```diff
@@ -14,46 +14,107 @@
 limitations under the License.
 """
 
-"""Integration tests for test_checkpointing.sh"""
+"""
+Integration tests for test_checkpointing.sh
+
+Note: Make sure to run
+  `bash setup_gcsfuse.sh DATASET_GCS_BUCKET=gs://maxtext-dataset MOUNT_PATH=/tmp/gcsfuse/`
+before running tests locally.
+"""
+
 from datetime import datetime
-import subprocess
+import json
+from math import isclose
 import os.path
 import pytest
 from MaxText.globals import PKG_DIR
-from MaxText.tests.globals import TEST_DISABLE_SUBPROCESS, TEST_DISABLE_SUBPROCESS_STR
+from MaxText.train import main as train_main
```

Comment:
Not for this PR, but it would be nice if we could clean up this import in the future to something more like:

```
from MaxText.train import train
```